index.js

```diff
@@ -7,7 +7,11 @@ const client = new Discord.Client();
 client.on('message', message => {
   switch(message.content){
     case '!bump':
+<<<<<<< Updated upstream
       message.channel.send('BUMP');
+=======
+      message.channel.send('Don't be a party pooper, send a message!!);
+>>>>>>> Stashed changes
```

Comment:
Ah, now it has conflicts. This is because your branch was out of date locally, you'll have to resolve the conflicts first.

Reply:
So what do I have to do to resolve that??

Reply:
Well you can resolve the conflicts manually by changing the file or with a tool. To prevent this from happening in the future, use git pull before pushing changes so you have the latest version.